### PR TITLE
Get creative content

### PR DESCRIPTION
**Pull request by jonkeller**
Clone PR #13069 because tests failing due to merge problems. Please continue previous review here.

Allows amp-analytics request to include htmlAttr expansion variable. Example:
${htmlAttr(img,src,width)}
...to get the src and width attrs of images. Documentation to follow in separate PR.
Caveats:

Only up to 10 matching elements will be returned.
An element counts as a match if it matches the CSS query selector (first arg) and has one or more of the attrs specified in the remaining args

### CODE REVIEW

File: extensions/amp-analytics/0.1/sandbox-vars-whitelist.js [hunk 1]
```diff
@@ -48,4 +48,5 @@ export const SANDBOX_AVAILABLE_VARS = {
   'FIRST_CONTENTFUL_PAINT': true,
   'FIRST_VIEWPORT_READY': true,
   'MAKE_BODY_VISIBLE': true,
+  'HTML_ATTR': true,
```
Comment (lannka) [inline extensions/amp-analytics/0.1/sandbox-vars-whitelist.js]: is this change needed?

File: extensions/amp-analytics/0.1/amp-analytics.js [hunk 2]
```diff
@@ -720,13 +729,55 @@ export class AmpAnalytics extends AMP.BaseElement {
...
+   * this one needs access to this.win.document, which if the amp-analytics
+   * tag is contained within an amp-ad tag will NOT be the parent/publisher
+   * page. Hence the need to put it here.
+   * @param {string} cssSelector Elements matching this selector will be
+   *     included, provided they have at least one of the attributeNames
+   *     set, up to a max of 10.
+   * @param attributeNames The attributes whose values will be returned
+   * @returns {string}
+   */
+  htmlAttrBinding_(cssSelector, ...attributeNames) {
```
Comment (lannka) [inline extensions/amp-analytics/0.1/amp-analytics.js]: would it be possible to move it to A4AVariableSource.

File: extensions/amp-analytics/0.1/sandbox-vars-whitelist.js (hunk 1 above)
Comment (jonkeller) [reply extensions/amp-analytics/0.1/sandbox-vars-whitelist.js]: Removed locally and tested, no change. Will leave out.

File: extensions/amp-analytics/0.1/amp-analytics.js (hunk 2 above)
Comment (jonkeller) [reply extensions/amp-analytics/0.1/amp-analytics.js]: @lannka  I'm working on this, but it would require an exception to the dependency rules. Is that ok?

**Comment by JoelRegus** [comment]
@jonkeller I noticed that this works in my vendors.js file:
```
${htmlAttr(img,src,width)}
```

But this fails:
```
${htmlAttr(#img,src,width)}
```

Which is a valid css selector. 

This also fails:
```
${htmlAttr("img",src,width)}
```

Which is a common mistake I think people are likely to make.

**Comment by jonkeller** [comment]
Thanks, @JoelRegus! I'll investigate those.

**Comment by jonkeller** [comment]
This PR is stale. Closing in favor of #13671.
